src/vellum/workflows/workflows/base.py

```diff
@@ -680,27 +680,39 @@ def deserialize_state(
 
     @staticmethod
     def load_from_module(module_path: str) -> Type["BaseWorkflow"]:
-        workflow_path = f"{module_path}.workflow"
-        module = importlib.import_module(workflow_path)
-        workflows: List[Type[BaseWorkflow]] = []
-        for name in dir(module):
-            if name.startswith("__"):
-                continue
-
-            attr = getattr(module, name)
-            if (
-                inspect.isclass(attr)
-                and issubclass(attr, BaseWorkflow)
-                and attr != BaseWorkflow
-                and attr.__module__ == workflow_path
-            ):
-                workflows.append(attr)
-
-        if len(workflows) == 0:
-            raise ValueError(f"No workflows found in {module_path}")
-        elif len(workflows) > 1:
-            raise ValueError(f"Multiple workflows found in {module_path}")
-        return workflows[0]
+        from vellum.workflows.exceptions import WorkflowInitializationException
+
+        try:
+            workflow_path = f"{module_path}.workflow"
+            module = importlib.import_module(workflow_path)
+            workflows: List[Type[BaseWorkflow]] = []
+            for name in dir(module):
+                if name.startswith("__"):
+                    continue
+
+                attr = getattr(module, name)
+                if (
+                    inspect.isclass(attr)
+                    and issubclass(attr, BaseWorkflow)
+                    and attr != BaseWorkflow
+                    and attr.__module__ == workflow_path
+                ):
+                    workflows.append(attr)
+
+            if len(workflows) == 0:
+                raise ValueError(f"No workflows found in {module_path}")
+            elif len(workflows) > 1:
+                raise ValueError(f"Multiple workflows found in {module_path}")
+            return workflows[0]
+        except TypeError as e:
```

Comment:
wrap this `try/except` only around `module = importlib.import_module(workflow_path)`